src/pages-conditional/wallets/find-wallet.js

```diff
@@ -87,6 +86,14 @@ const Header = styled.header`
   }
 `
 
+const InfoBannerContainer = styled.div`
```

Comment:
Why not just style `<InfoBanner/>` directly vs. creating this container?

Reply:
It makes the banner go full width as it's not in a parent container with any padding/margin set.